index.html

```diff
@@ -18,153 +18,181 @@
+                        <p>Back End Dev</p>
+                        <img src="img/counter.png" alt="counter">
+                        <p>2015</p>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <ul class="tags">
+                        <li class="tag">html</li>
+                        <li class="tag">css</li>
+                        <li class="tag">javascript</li>
+                    </ul>
+                    <a href="#" class="work-link">See Project</a>
                 </div>
-                <button type="button" class="button">See Project</button>
             </div>
-        </section>
-        <section class="cardworks">
-            <div class="container">
-                <div class="workimage"><img src="img/Snapshoot2.png" alt="snapshoot"></div>
-                <h1 class="greet">Multi-Post</h1>
-                <h1 class="greet">Stories</h1>
-                <div class="frame2">
-                    <div class="client">CANOPY</div>
-                    <div><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
-                    <div class="role">Back End Dev</div>
-                    <div><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
-                    <div class="role">2015</div>
+        </div>
+        <div class="work-container">
+            <div class="work">
+                <div class="work-details">
+                    <h2 class="work-title">Multi-Post Stories</h2>
+                    <div class="work-subtitle">
+                        <p class="company">FACEBOOK</p>
+                        <img src="img/counter.png" width="10" height="10" alt="counter">
+                        <p>Back End Dev</p>
+                        <img src="img/counter.png" alt="counter">
+                        <p>2015</p>
+                    </div>
+                    <p class="work-description">Experimental content creation feature that allows users to add to an
+                        existing story over the course of a day without spamming their friends..</p>
+                    <ul class="tags">
+                        <li class="tag">html</li>
+                        <li class="tag">css</li>
+                        <li class="tag">javascript</li>
+                    </ul>
+                    <a href="#" class="work-link">See Project</a>
                 </div>
-                <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <div class="tags">
-                    <p class="tag">html</p>
-                    <p class="tag">css</p>
-                    <p class="tag">javascript</p>
+                <div class="img-container">
+                    <img src="img/card2.png" class="work-image" alt="card2">
                 </div>
```

Comment:
You seem to have modified your image links. Kindly crosscheck and add the proper links so they can display normally.

![image](https://user-images.githubusercontent.com/20554583/187015485-f0cc6e5c-cf50-4588-8226-d0687665bf25.png)